app/src/main/java/org/p2p/wallet/utils/ImageViewExtensions.kt

```diff
@@ -0,0 +1,14 @@
+package org.p2p.wallet.utils
+
+import android.widget.ImageView
+import androidx.annotation.DrawableRes
+import androidx.core.view.isVisible
+
+infix fun ImageView.withImageOrGone(@DrawableRes imageRes: Int?) {
```

Comment:
I am not agreeing to have and use this extension, it's so implicit. I can understand to use smth liek `isGoneOrVisible` with boolean param because boolean has only two values. But Int has 2mil values + null, so from the signature I can't understand what should I pass to this extension to make image view gone. 
can you imagine this usage:
```kotlin 
myView.withImageOrGone(null)
```

Reply:
you do not need to use this extension to make ImageView gone like this
you need to use imageView.isVisible = false
this extension is only for cases when you need to use resInt and it could be Int or null
not to write such code like

imageRes = getImageRes;
if (imageRes == null) {
  imageview.isVisible = false
} else {
  imageView.setImageResource(imageRes)
}

just for this case
i believe this cases will be not only here